src/main/java/de/dhbw/karlsruhe/domain/ports/dialogs/DifficultySelectionCliPort.java

```diff
@@ -0,0 +1,7 @@
+package de.dhbw.karlsruhe.domain.ports.dialogs;
+
+public interface DifficultySelectionCliPort {
+
+    void writeNoEqualDifficulty();
```

Comment:
Ich denke bei den Port/Adapter Methoden reicht es auch, wenn man überall das "write" raus nimmt. Klingt sonst wieder stark auf Cli bezogen und denke es ist auch ohen das "write" eindeutig was gemacht wird, dadurch da das ja immer in der Form OutputPort.NoEqualDifficulty() im Code genutzt wird,

Reply:
Bin ich tatsächlich anderer Meinung. Denke, dass wenn jemand die Funktionen nutzen möchte, macht writeXYZ() schon von der Benennung her Sinn. Das ist eben eindeutig und man weiß sofort, was die Funktion macht. Falls jemand von außen die Funktion aufruft mit "outputPort.NoEqualDifficulty()" kann man sich denke ich nicht 100% sicher sein, was mit output gemeint ist.

Reply:
Habe die Methode umbenannt und angepasst.